src/app/components/CardBorder.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from "react";
+
+export default function CardBorder(props: { children: ReactNode }) {
```

Comment:
```suggestion
export interface CardBorderProps {
  children: ReactNode;
}

export default function CardBorder(props: CardBorderProps) {
```

for consistency with other component props definitions